### DISCUSSION TITLE
Can I set Laravel Pint as a formatter?

### DISCUSSION THREAD

**Discussion by chimit** [open-southeners/vscode-laravel-pint#26] [Q&A] [answered]
I would like to be able to choose a formatter. Can I set Pint as a default formatter?

<img width="214" alt="image" src="https://user-images.githubusercontent.com/839349/190843798-c0bd93bf-94ee-43fe-b575-361d66b72fc6.png">

**Comment by d8vjork** [comment]
Which version of the extension do you have?

v0.x = not using formatters API of VSCode
v1.x (pre-release) = using VSCode API

**Comment by chimit** [comment]
v0.7.3 from the VS Code Marketplace. How do I install the v1?

**Comment by d8vjork** [comment, marked as answer]
<img width="475" alt="image" src="https://user-images.githubusercontent.com/2331052/190895774-b200fdad-1f56-49fd-83f9-adccbfd8a5c7.png">

You need to install the pre-release version clicking to the button at the right
